torch/export/_trace.py

```diff
@@ -1465,20 +1590,27 @@ def _export(
 
     # Call the appropriate export function based on the strictness of tracing.
     export_func = _strict_export if strict else _non_strict_export
-    export_artifact = export_func(
-        mod,
-        args,
-        kwargs,
-        dynamic_shapes,
-        preserve_module_call_signature,
-        pre_dispatch,
-        original_state_dict,
-        orig_in_spec,
-        _allow_complex_guards_as_runtime_asserts,
-        _disable_forced_specializations,
-        _is_torch_jit_trace,
-    )
 
+    # TODO we only want to do this for aot-dispatch for now. Pre-dispatch will be deleted soon
```

Comment:
wdym pre-dispatch will be deleted soon? 😛

Reply:
Yep should be more clear here! I meant we are deleting pre-dispathc functional export.